expressEngine/lib/errorHandler/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+import error from '../../cms/common';
+
+export default function errorHandler(err, req, res) {
+  const { message, status } = err;
```

Comment:
@chiragSuccessive Please set a default status, in case there is no status provided.